src/static/js/routes/offices/index.js

```diff
@@ -15,11 +15,9 @@ function init() {
     return;
   }
 
-  var breakpointPx = 599;
-
   $( '.expandable__mobile-only' ).each( function() {
     // ESLint no-new rule ignored
-    new MobileOnlyExpandable( $( this ), breakpointPx ); // eslint-disable-line
+    new MobileOnlyExpandable( $( this ) ); // eslint-disable-line
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/eeeafcc1078652feeca68447f2d2bc47f2216736